puppet/modules/jenkins_job_builder/files/theforeman.org/scripts/test/test_develop.sh

```diff
@@ -24,6 +24,9 @@ while ! bundle install --without=development -j 5; do
   fi
 done
 
+# run npm install if package.json exists
+[ -e "$APP_ROOT/package.json"] && npm install
```

Comment:
Since integration tests only run on postgres, I used this for the pull request test:

```
[ ${database} = postgresql ] && [ -e "$APP_ROOT/package.json" ] && npm install
```